app/sprinkles/admin/templates/tables/users.html.twig

```diff
@@ -72,25 +73,27 @@
                 {{/ifx }}
             {{/ifx }}
         >
-            <div class="btn-group">
-                {{#ifx row.flag_enabled '==' 0 }}
-                    <button type="button" class="btn btn-default dropdown-toggle" data-toggle="dropdown">
-                        {% endverbatim %}{{translate("DISABLED")}}{% verbatim %}
-                        <span class="caret"></span>
-                    </button>
+            {{#ifx row.flag_enabled '==' 0 }}
+                <span class="text-muted">
+                    {% endverbatim %}{{translate("DISABLED")}}{% verbatim %}
+                </span>
+            {{ else }}
+                {{#ifx row.flag_verified '==' 0 }}
+                    <span class="text-yellow">
+                        {% endverbatim %}{{translate("UNACTIVATED")}}{% verbatim %}
+                    </span>
                 {{ else }}
-                    {{#ifx row.flag_verified '==' 0 }}
-                        <button type="button" class="btn btn-warning dropdown-toggle" data-toggle="dropdown">
-                            {% endverbatim %}{{translate("UNACTIVATED")}}{% verbatim %}
-                            <span class="caret"></span>
-                        </button>
-                    {{ else }}
-                        <button type="button" class="btn btn-primary dropdown-toggle" data-toggle="dropdown">
-                            {% endverbatim %}{{translate("ACTIVE")}}{% verbatim %}
-                            <span class="caret"></span>
-                        </button>
-                    {{/ifx }}
+                    <span>
+                        {% endverbatim %}{{translate("ACTIVE")}}{% verbatim %}
+                    </span>
                 {{/ifx }}
+            {{/ifx }}
+        </td>
+    </script>
+    <script id="user-table-column-actions" type="text/x-handlebars-template">
+        <td>
+            <div class="btn-group">
+                <button type="button" class="btn btn-primary dropdown-toggle" data-toggle="dropdown">Edit <span class="caret"></span></button>
```

Comment:
This https://github.com/userfrosting/UserFrosting/blob/master/app/sprinkles/core/locale/en_US/messages.php#L99 should be used here.

Reply:
Thanks.